guide/reindex-wallet.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Reindexing your Firo wallet"
+summary: 
+tags: [guide]
+author: "Anwar P."
+permalink: /guide/reindex-wallet.html
+---
+## Reindexing your wallet
+
+If you have trouble syncing your wallet and it appears to be stuck on a block, please start it again with the **-reindex-chainstate flag**. This will start the Zcoin wallet and begin the blockchain reindex process and **will take several hours**. 
+
+If you suspect corrupted blocks, please run **\-reindex** instead. 
+
+Make sure you are always on the latest release by going to our [release page](https://github.com/zcoinofficial/zcoin/releases).
```

Comment:
Make sure you are always on the latest release by going to our [release page](https://github.com/firoorg/firo/releases).

Reply:
Fixed